bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```
@@ -80,6 +80,11 @@ spec:
       {{- end }}
       hostNetwork: {{ .Values.postgresql.hostNetwork }}
       hostIPC: {{ .Values.postgresql.hostIPC }}
+      {{- if .Values.postgresql.tls.enabled }}
+      initContainers:
+      {{- else if and .Values.volumePermissions.enabled (or (or (not (empty .Values.postgresql.extendedConf)) (not (empty .Values.postgresql.extendedConfCM))) .Values.persistence.enabled) }}
+      initContainers:
+      {{- else if or .Values.postgresql.initContainers .Values.postgresql.extraInitContainers }}
```

Could you please reduce this to a single conditional?

```suggestion
{{- if or .Values.postgresql.tls.enabled .Values.postgresql.initContainers .Values.postgresql.extraInitContainers (and .Values.volumePermissions.enabled (or (or (not (empty .Values.postgresql.extendedConf)) (not (empty .Values.postgresql.extendedConfCM))) .Values.persistence.enabled)) }}
```